### ISSUE TITLE
Not working on gtk4 apps after update to gnome 47!

### ISSUE THREAD

**Issue by sentakuhm** [phisch/phinger-cursors#28]
after clean install of gnome 47 and hyprland cursor theme not working on gtk4 apps.
i make sure settings are correct:
`gsettings set org.gnome.desktop.interface cursor-theme 'phinger-cursors-light'`
and use `nwg-look` tool also, else everything working well like gtk3 apps...

Edit: tested other cursor theme like `Bibata` and its working fine on gtk4.

**Comment by Jappie3** [comment]
I saw on discord that you opened an issue, but this is phisch/phinger-cursors - I think you meant to open this in Jappie3/hyprcursor-phinger.

To answer your question: the hyprcursor theme I created with phinger-cursors never worked on GTK apps to begin with, as it is a Hyprcursor theme. GTK does not support server-side cursors, and thus Hyprcursor themes cannot work on GTK apps. See https://wiki.hyprland.org/Hypr-Ecosystem/hyprcursor/#important-notes for more information.

Until GTK supports server-side cursors, you'll need to install phinger-cursors as an XCursor theme as well to make it work consistently (which is probably what you did initially and what broke with the update).
